docs/content/latest/admin/yb-ts-cli.md

```diff
@@ -0,0 +1,229 @@
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### current_hybrid_time
+
+Prints the value of the current [hybrid time](../../../architecture/transactions/single-row-transactions/#hybrid-time-as-an-mvcc-timestamp).
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] current_hybrid_time
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### delete_tablet
+
+Deletes the tablet with the specified tablet ID (`tablet_id`) and reason.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] delete_tablet <tablet_id> "<reason-string>"
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- *tablet_id*: The identifier (ID) for the tablet.
+- *reason-string*: Text string providing useful information on why the tablet was deleted.
+
+##### dump_tablet
+
+Dump, or export, the specified tablet ID (`tablet_id`).
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] dump_tablet <tablet_id>
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- *tablet_id*: The identifier (ID) for the tablet.
+
+##### list_tablets
+
+Lists the tablets on the specified tablet server, displaying the following information:
+
+- Tablet id
+- State
+- Table name
+- Partition
+- Schema
+- properties
+  - lumn_name
+  - Tablet id
+  - State
+  - Table name
+  - Partition
+  - Schema
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] list_tablets
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### set_flag
+
+Sets the specified configuration option (flag) for the tablet server.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli [ --server_address=<host>:<port> ] set_flag [ --force ] <flag> <value>
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- --force: Flag to to allow a change to a flag (option) that is not explicitly marked as runtime-settable. The change may be ignored on the server or may cause the server to crash. See [--force](#force).
+- *flag*: The `yb-tserver` configuration option (without the `--` prefix) to be set. See [`yb-tserver`](../../reference/configuration/yb-tserver/#configuration-options)
+- *value*: The value to be applied.
+
+{{< note title="Important" >}}
+
+The `set_flag` command changes the in-memory value of the specified flag, atomically, for a running process and can alter its behavior.  **The change does NOT persist across restarts.**
+
+In practice, there are some flags (options) that are runtime safe to change (runtime-settable) and some that are not. These are generally denoted probably only in the code, for now. For example, the bind address of the server cannot be changed at runtime, since the server binds just once at startup.
```

Comment:
> These are generally denoted probably only in the code

Not sure if this is useful to have in the docs :) sounds more confusing then useful..

Maybe instead we can explicitly mark them in the configuration flag section of our docs and call them out as the ones NOT runtime changeable. Obviously there might be some lag between code and doc changes, but I find it better to tell users to look at some other docs location, then tell them we might not have this info anywhere easily accessible for them.

Reply:
I still don't like this part `These are generally denoted probably only in the code, for now.` I'd rather this say nothing at all about this and instead just put a WIP note on the configuration flag page.

Reply:
Removed that sentence. Added to note in `yb-ts-cli set_flag` that just added to the note in set_flags that most flags are probably runtime-settable, but that we need to review and update the docs accordingly